### PR TITLE
update mdsd 1.25.0

### PR DESCRIPTION
**Pull request by pfrcks**

### CODE REVIEW

**Comment by pfrcks** [comment]
verified that syslog data from mdsd.qos shows up in application insights traces table

File: kubernetes/linux/setup.sh [hunk 1]
```diff
@@ -33,9 +33,9 @@ sed -i -e 's/# en_US.UTF-8 UTF-8/en_US.UTF-8 UTF-8/' /etc/locale.gen && \
 
 #install oneagent - Official bits (3/14/2022)
 if [ "${ARCH}" != "arm64" ]; then
-    wget "https://github.com/microsoft/Docker-Provider/releases/download/1.17.0/azure-mdsd_1.17.0-build.master.354_x86_64.deb" -O azure-mdsd.deb
+    wget "https://github.com/microsoft/Docker-Provider/releases/download/official%2Fmdsd%2F1.25.0/azure-mdsd_1.25.0-build.master.476_x86_64.deb" -O azure-mdsd.deb
 else
-    wget "https://github.com/microsoft/Docker-Provider/releases/download/1.17.1-arm64-master/azure-mdsd_1.17.1-build.master.366_aarch64.deb" -O azure-mdsd.deb
+    wget "https://github.com/microsoft/Docker-Provider/releases/download/official%2Fmdsd%2F1.25.0/azure-mdsd_1.25.0-build.master.476_aarch64.deb" -O azure-mdsd.deb
```
Comment (ganga1980) [inline kubernetes/linux/setup.sh:38]: is this being validated on Arm64 and x64 on both legacy and msi auth?

Comment (pfrcks) [reply kubernetes/linux/setup.sh:38]: yes

**Comment by jatakiajanvi12** [comment]
Any change in performance?

**Comment by pfrcks** [comment]
> Any change in performance?

I am not seeing any notable deviations

**Review by jatakiajanvi12 — APPROVED** [review]

**Pull request merged**